CHANGELOG.md

```diff
@@ -5,12 +5,16 @@
 * Updated to Realm Sync v1.0.4.
 * Updated to Realm Core v2.3.1.
 
+### Enhancements
+
+* Improved performance of getters and setters in proxy classes. As a side-effect of this improvement, now all proxy classes have a public method of `realm$injectObjectContext()`.
```

Comment:
No reason to name the public method I think? Proxy classes are not visible anyway?